### DISCUSSION TITLE
Screen not waking when lid is closed in Mac

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5894] [Issues]
Hi all,

I'm not sure if it relates to the Mesh but I'll try.
When I run the Display Sleep it works but Display Awake not working when the lid is closed.
It occurs also when I run sudo _pmset -a disablesleep 1_ to disable sleep
It works when lid in open so I guess it's a funktion in the Mac itself.
Does the Mac has a config file like in Linux Mint where you can disable the LidSwitch or something similar?
Thanks.

**Comment by dinger1986** [comment]
There's an app called insomnia or similar for the mac or at least was it was made for this

**Reply by ghost** [reply]
Hi, you mean InsomniaX.
When I run Display Sleep and then run Display Awake with the lid closed the issue still occurs also with Insomnia app running.
It's like the the Display Sleep overrides Insomnia or any other pmset command.
I have a workaround where I'm just lowering brightness to 0 but it's not a solution.
Thanks,

**Reply by dinger1986** [reply]
Ah ok

**Comment by si458** [comment]
the is currently an issue where if the mac screens go to sleep, and then you connect to the mac,
meshcentral doesnt always register the fact the screens awake.
this generally happens if you have the 'lock screen with password' set in OSX where you need to enter ya password after the monitor goes to sleep

ive had some success with running the command 'caffeinate -d -t 2' to wake ths screen up, BUT you need to disconnect and reconnect over n over to then get it to display the login screen

not sure if this helps

**Reply by ghost** [reply]
I don't a Lock Screen with Password and just for reference I sometime uses DWservice remote and it works there.
Will followup on this one if I find anything or if there is any update.
Thanks.
